packages/util-dynamodb/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@aws-sdk/util-dynamodb",
+  "version": "1.0.0-gamma.0",
+  "scripts": {
+    "prepublishOnly": "yarn build:cjs && yarn build:es",
+    "pretest": "yarn build:cjs",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build": "yarn build:es && yarn build:cjs",
+    "test": "jest"
+  },
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "types": "./dist/cjs/index.d.ts",
+  "author": {
+    "name": "AWS SDK for JavaScript Team",
+    "url": "https://aws.amazon.com/javascript/"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@aws-sdk/client-dynamodb": "1.0.0-gamma.9",
```

Comment:
If only TS interface is imported from `@aws-sdk/client-dynamodb`, I think we can move it to dev dependency

Reply:
Fixed in [`2e12934` (#1531)](https://github.com/aws/aws-sdk-js-v3/pull/1531/commits/2e12934268857e9b19b81c20eb41f25962c2221d)